packages/core-bridge/index.d.ts

```diff
@@ -110,13 +110,12 @@ export interface PrometheusMetricsExporter {
 
 /**
  * Metrics exporters supported by Core
+ *
+ * `temporality` is the type of aggregation temporality for metric export. Applies to both Prometheus and OpenTelemetry exporters.
+ *
+ * See the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/ce50e4634efcba8da445cc23523243cb893905cb/specification/metrics/datamodel.md#temporality) for more information.
```

Comment:
Why?

Reply:
previous comment location doesn't show up in api ref:

![image](https://user-images.githubusercontent.com/251288/192897269-82275cfa-5f09-4119-80c6-da6fe4a91a2f.png)

Reply:
:(